File: dataset/output-bk/CLAUDE_claude-3-5-sonnet-20240620/Task31_CLAUDE_claude-3-5-sonnet-20240620.cpp

#include <iostream>
#include <string>

std::string make_complement(std::string dna) {
    for (char& c : dna) {
        switch (c) {
            case \'A\':
                c = \'T\';
                break;
            case \'T\':
                c = \'A\';
                break;
            case \'C\':
                c = \'G\';
                break;
            case \'G\':
                c = \'C\';
                break;
        }
    }
    return dna;
}

int main() {
    std::cout << make_complement("ATTGC") << std::endl; // Output: TAACG
    std::cout << make_complement("GTAT") << std::endl;  // Output: CATA
    return 0;
}

File: dataset/output-bk/CLAUDE_claude-3-5-sonnet-20240620/Task31_CLAUDE_claude-3-5-sonnet-20240620.java

public class DnaStrand {
    public static String makeComplement(String dna) {
        StringBuilder complement = new StringBuilder();
        for (char c : dna.toCharArray()) {
            switch (c) {
                case \'A\':
                    complement.append(\'T\');
                    break;
                case \'T\':
                    complement.append(\'A\');
                    break;
                case \'C\':
                    complement.append(\'G\');
                    break;
                case \'G\':
                    complement.append(\'C\');
                    break;
            }
        }
        return complement.toString();
    }

    public static void main(String[] args) {
        System.out.println(makeComplement("ATTGC")); // Output: TAACG
        System.out.println(makeComplement("GTAT"));  // Output: CATA
    }
}
